1-exercises/A-accessing-values/exercise3.js

```diff
@@ -22,6 +22,12 @@ let basketballTeam = {
 
 // write code here
 
+let topPlayers = basketballTeam.topPlayers;
+topPlayers.sort();
+for (let player of topPlayers) {
+  console.log(player);
+}
+
 
 /* EXPECTED RESULT
```

Comment:
Here you could make it easy by finding the index like this `console.log(basketballTeam.topPlayers[index number ]);
`